metadata/email_addressesMetaData.php

```diff
@@ -80,8 +80,12 @@
             'default'        => 0,
             'vname'         => 'LBL_OPT_OUT',
         ),
-            
-                'confirm_opt_in' => array(
+        'opt_in_email_created' => array(
```

Comment:
Where and why did you use this `opt_in_email_created` for? I don't find any reference to this in your PR?

Reply:
It isn't used in this PR. In my next pr we will use it.

Reply:
"It isn't used in this PR. In my next pr we will use it." - Why not implementing this in that PR then? Or finished the task here? - Sorry but I disagree, I think totally pointless and confusing to implementing a half task here an other one half of same task in an other PR...